airbyte-integrations/connectors/source-mysql-v2/src/main/kotlin/io/airbyte/integrations/source/mysql/MysqlJdbcEncryption.kt

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.source.mysql
+
+import io.airbyte.cdk.ConfigErrorException
+import io.airbyte.cdk.SystemErrorException
+import io.airbyte.cdk.jdbc.SSLCertificateUtils
+import io.github.oshai.kotlinlogging.KotlinLogging
+import java.net.MalformedURLException
+import java.net.URI
+import org.apache.commons.lang3.RandomStringUtils
+
+private val log = KotlinLogging.logger {}
+
+class MysqlJdbcEncryption(
+    val sslMode: SSLMode = SSLMode.PREFERRED,
+    val caCertificate: String? = null,
+    val clientCertificate: String? = null,
+    val clientKey: String? = null,
+    val clientKeyPassword: String? = null,
+) {
+    companion object {
+        const val TRUST_KEY_STORE_URL: String = "trustCertificateKeyStoreUrl"
+        const val TRUST_KEY_STORE_PASS: String = "trustCertificateKeyStorePassword"
+        const val CLIENT_KEY_STORE_URL: String = "clientCertificateKeyStoreUrl"
+        const val CLIENT_KEY_STORE_PASS: String = "clientCertificateKeyStorePassword"
+        const val CLIENT_KEY_STORE_TYPE: String = "clientCertificateKeyStoreType"
+        const val TRUST_KEY_STORE_TYPE: String = "trustCertificateKeyStoreType"
+        const val KEY_STORE_TYPE_PKCS12: String = "PKCS12"
+        const val SSL_MODE: String = "sslMode"
+    }
+
+    private fun getOrGeneratePassword(): String {
+        if (!clientKeyPassword.isNullOrEmpty()) {
+            return clientKeyPassword
+        } else {
+            return RandomStringUtils.randomAlphanumeric(10)
```

Comment:
if this is all you need the `commons-lang3` package for, remove the dependency and just generate a random UUID instead

Reply:
done. thanks!